src/main/java/com/unstoppabledomains/resolution/naming/service/uns/UNSInternal.java

```diff
@@ -123,10 +128,9 @@ public  String getOwner(String domain) throws NamingServiceException {
   private Thread parseRegistryThread(String contractName, String address, List<String> toPopulateList) {
     return new Thread(() -> {
       String registryAddress = NetworkConfigLoader.getContractAddress(chainId, contractName);
-      String deploymentBlock = NetworkConfigLoader.getDeploymentBlock(chainId, contractName);
       Registry registryContract = new Registry(blockchainProviderUrl, registryAddress, provider);
        try {
-        List<String> tokensFromRegistry = registryContract.getTokensOwnedBy(address, deploymentBlock);
+        List<String> tokensFromRegistry = registryContract.getTokensOwnedBy(address, "earliest");
```

Comment:
Why not use the deploymentBlock we have from the config? in a case when deploymentBlock is 0x0 getDeploymentBlock from NetworkConfigLoader will return 'earliest'.

Reply:
Thanks for catching! I forgot about this. This was a temporary change along with the polygon provider for testnet.

I ran into some issues with the temporary polygon API provider. It requires hex block numbers without leading zeros.
We can save deployment blocks in `uns-config` in the correct format, once we decide which provider to use. 

I've updated the comment about changes that were made for the temporary API